build/localnet/Tiltfile

```diff
@@ -2,7 +2,8 @@
 load("ext://helm_resource", "helm_resource", "helm_repo")
 load("ext://namespace", "namespace_create")
 load("ext://restart_process", "docker_build_with_restart")
-load('ext://tests/golang', 'test_go')
+load("ext://tests/golang", "test_go")
+load("ext://syncback", "syncback")
```

Comment:
This broke my localnet because I didn't have an update to date (>3.0.0 or greater) version of rsync. The dependency should be listed in the prerequisites section of the development setup guide.
More info at [tilt-extensions/syncback - Usage](https://github.com/tilt-dev/tilt-extensions/tree/master/syncback#usage)

Reply:
[`489978f` (#874)](https://github.com/pokt-network/pocket/pull/874/commits/489978f040cffab5b146014ecaf12222f5828e70)